lms/djangoapps/labster_course_license/utils.py

```diff
@@ -105,8 +106,13 @@ def course_tree_info(store, simulations, licensed_simulations):
     Retuns information about the course's xblocks.
     """
     course_info = {}
+    invalid_simulations = []
+    validation_pattern = r'^https:\/\/[\w\d]*\.labster\.com\/[a-z]*\/[\w\d]*\/$'
```

Comment:
it looks like we're using `[a-zA-Z0-9]+` and you cannot use `*`. See https://github.com/Livit/Livit.Learn.LTI/blob/master/django_project/core/urls.py#L12